src/sleep/sleep.c

```diff
@@ -264,10 +264,18 @@ static int execute(
 
 static int execute_s2h(const SleepConfig *sleep_config) {
         _cleanup_hashmap_free_ Hashmap *last_capacity = NULL, *current_capacity = NULL;
+        _cleanup_(freeze_thaw_user_slice) const char *method = "thaw";
+        const char *method_freeze = "freeze";
         int wakeup_type_bit = 0, r;
 
         assert(sleep_config);
 
+        r = freeze_thaw_user_slice(&method_freeze);
+        if (r <= 0) {
```

Comment:
log_debug_errno will assert if `r` is 0. But it's not necessary to error out if it is, as that's an expected condition, so just check `r < 0`